### PR TITLE
Added Node QPS test

### PR DESCRIPTION
**Pull request by murgatroid99**
I just wanted this so I could test other changes.

### CODE REVIEW

File: src/node/examples/qps_test.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+var interop_server = require('../interop/interop_server.js');
+
+function runTest(concurrent, seconds, callback) {
+  var testServer = interop_server.getServer(0, false);
+  testServer.server.listen();
+  var client = new testProto.TestService('localhost:' + testServer.port);
+
+  var warmup_num = 100;
+
+  async.waterfall([
```
Comment (tbetbetbe) [inline src/node/examples/qps_test.js]: for each func used in this async.waterfall block
- make it a func at the same level as the async.waterfall call
- add a doc comment that provides a short explanation what it does

File: src/node/examples/qps_test.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+'use strict';
+
```
Comment (tbetbetbe) [inline src/node/examples/qps_test.js]: provide a file-level comment explaining what the code in this file does.

- it should also include usage instructions, even it that's a simple one-liner

Please read through [the relevant Javascript Style guide section](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Comments) and update any other sections of this file that lack the necessary comments.

File: src/node/examples/qps_test.js (hunk 1 above)
Comment (murgatroid99) [inline src/node/examples/qps_test.js]: done

File: src/node/examples/qps_test.js (hunk 2 above)
Comment (murgatroid99) [inline src/node/examples/qps_test.js]: done

**Pull request merged**